tuf/signed/interface.go

```diff
@@ -47,7 +39,6 @@ type KeyService interface {
 // CryptoService defines a unified Signing and Key Service as this
```

Comment:
This comment is no longer true.  Also, not necessary in this PR, but should `CryptoService` be eliminated in favor of just `KeyService`, or do we anticipate other uses for `CryptoService`?